malaria24/ona/tests/test_admin.py

```diff
@@ -55,7 +55,7 @@ def test_setting_disables_send_to_jembi(self, mock_task):
         }
         list_url = reverse('admin:ona_reportedcase_changelist')
         response = self.client.post(list_url, data, follow=True)
-        mock_task.not_called()
+        self.assertFalse(mock_task.called)
```

Comment:
Related to this pull?

Reply:
@bearnard more of a fix to the tests than related to the addition of the postgres versions.
But this was an error that was causing the tests to fail. There's no record of the tests passing previously so /shrug